modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/service/impl/CPDefinitionInventoryLocalServiceImpl.java

```diff
@@ -207,12 +222,55 @@ public CPDefinitionInventory updateCPDefinitionInventory(
 		cpDefinitionInventory.setBackOrders(backOrders);
 		cpDefinitionInventory.setMinOrderQuantity(minOrderQuantity);
 		cpDefinitionInventory.setMaxOrderQuantity(maxOrderQuantity);
+
+		_validateAllowedOrderQuantities(
+			allowedOrderQuantities, cpDefinitionInventory.getGroupId());
+
 		cpDefinitionInventory.setAllowedOrderQuantities(allowedOrderQuantities);
 		cpDefinitionInventory.setMultipleOrderQuantity(multipleOrderQuantity);
 
 		return cpDefinitionInventoryPersistence.update(cpDefinitionInventory);
 	}
 
+	private void _validateAllowedOrderQuantities(
+			String allowedOrderQuantities, long groupId)
+		throws PortalException {
+
+		if (Validator.isNull(allowedOrderQuantities)) {
+			return;
+		}
+
+		CommerceCatalog commerceCatalog =
+			_commerceCatalogLocalService.fetchCommerceCatalogByGroupId(groupId);
+
+		DecimalFormatSymbols symbols = new DecimalFormatSymbols(
+			LocaleUtil.fromLanguageId(
+				commerceCatalog.getCatalogDefaultLanguageId()));
+
+		char groupingSeparator = symbols.getGroupingSeparator();
+		char decimalSeparator = symbols.getDecimalSeparator();
```

Comment:
As per the discussion on Slack, the pattern should be fixed `###,##0.00`

Then we ask the user via a help message to input it on the admin side. In the front store and delivery APIs, we can use the `CommerceQuantityFormatter` to present it in the user's locale.